### PR TITLE
Feat/best worst season eb

### PR DESCRIPTION
**Pull request by EricBelongea**
This adds two helper methods with unit testing and the two methods Best_season worst_season.

### CODE REVIEW

File: lib/team.rb
Comment (EricBelongea) [inline lib/team.rb:1]: I did a merge conflict that I expected to have happened. We had our methods in the same location on the Team class. I moved your method to be just below the def initialize. 

Looking at it now I think I messed up and your method is missing an end statement. My fault.

File: lib/team.rb [hunk 1]
```diff
@@ -18,5 +18,58 @@ def team_info(team_id)
       link: "link"
     }
     team_info.transform_keys! { |k| key_map[k] }
+
```
Comment (EricBelongea) [inline lib/team.rb:21]: I messed up, think we need an end statement here on line 21. We can both do this locally if you're cool with the merge.

File: spec/team_spec.rb
Comment (EricBelongea) [inline spec/team_spec.rb:1]: I feel like expected is a cheesy way to test, but covers a good bit tbh.

Comment (zkevkev) [reply spec/team_spec.rb:1]: I think this is totally acceptable and agree that it has good coverage.

**Review by zkevkev — APPROVED** [review]
I saw these methods on our call and they look fantastic! Thank you for doing the unit testing and being characteristically thorough,

**Pull request merged**